react/components/Helmet.tsx

```diff
@@ -0,0 +1,3 @@
+import { Helmet } from 'react-helmet'
+
+export default Helmet
```

Comment:
Why this new file?

Reply:
Builder-Hub will only export types from individual files 🤷 

Added a comment to the file